skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -416,57 +450,63 @@ def test_inference(
 ):
     # test inference backend for classifier and regressor models. This test can
     # take a lot of time and be flaky.
-    client = HfApi()
-
-    repo_path = repo_path_for_inference
-    model = model_func()
-    model_path = repo_path / "model.pickle"
-
-    with open(model_path, "wb") as f:
-        pickle.dump(model, f)
-
-    version = metadata.version("scikit-learn")
-    init(
-        model=model_path,
-        requirements=[f'scikit-learn="{version}"'],
-        dst=destination_path,
-        task=task,
-        data=data.data,
-    )
+    config_path, file_format = config_json
+    if file_format != "pickle":
+        return
```

Comment:
I think instead of silently skipping the inference test, we should parameterize and only test for `pickle`, and when inference API starts working with `.skops`, we add it here.